lib/message.rb

```diff
@@ -0,0 +1,14 @@
+require 'rubygems'
+require 'dotenv'
+Dotenv.load
+require 'twilio-ruby'
+
+class Message
+
+  def send(number, text)
```

Comment:
You could inject the `Twilio::REST::Client` via DI here. That would save some stubbing pain